scripts/qa_crowbarsetup.sh

```diff
@@ -3260,6 +3260,8 @@ function addfloatingip
     nova floating-ip-create | tee floating-ip-create.out
     floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
     nova add-floating-ip "$instanceid" "$floatingip"
+    # flush arp tables in case this floating ip was recently used for something else
+    arp -d "$floatingip"
```

Comment:
@dirkmueller that should update the neighbours too since it's doing an "arping -A " through the qg interface. No idea why it doesn't work. Can I have a look at a broken setup or how can I reproduce?